docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -0,0 +1,39 @@
+# Copyright IBM Corporation 2017.
```

Comment:
Please keep the copyright notices in the NOTICE file as described in [the contributing guide](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/CONTRIBUTING.md).
(I realize it needs removing from the existing Dockerfile src too!)